keyboards/handwired/tractyl_manuform/5x6_right/rpi2040/config.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 Pascal Jaeger <pascal.jaeger@leimstift.de>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define CRC8_USE_TABLE
+#define CRC8_OPTIMIZE_SPEED
+
+/* Reset to bootloader */
+#define RESET_PIN                                  GP22
+#define RESET_PIN_DOUBLE_TAP_RESET_TIMEOUT         1000U
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED     GP25
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
+
+/* split settings */
+#define SPLIT_POINTING_ENABLE
+#define SPLIT_HAND_PIN GP28 // high = left, low = right
+
+/* serial configuration for split keyboard */
+#define SERIAL_PIO_USE_PIO0
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_TX_PIN  GP0
+#define SERIAL_USART_RX_PIN  GP1
+#define SERIAL_USART_TIMEOUT 100
+#define SERIAL_USART_SPEED   921600
+#define SERIAL_USART_PIN_SWAP
+
+/* i2c config for oleds */
+#define I2C_DRIVER       I2CD1
+#define I2C1_SDA_PIN     GP2
+#define I2C1_SCL_PIN     GP3
+#define I2C1_CLOCK_SPEED 400000
+
+/* SPI config */
+#define SPI_DIVER    SPID0
+#define SPI_SCK_PIN  GP18
+#define SPI_MOSI_PIN GP19
+#define SPI_MISO_PIN GP16
+
+/* pmw3360 config */
+#define POINTING_DEVICE_CS_PIN GP17
+#define PMW33XX_LIFTOFF_DISTANCE      0x0C
+#define POINTING_DEVICE_ROTATION_180
+#undef  ROTATIONAL_TRANSFORM_ANGLE
+#define ROTATIONAL_TRANSFORM_ANGLE    -40
```

Comment:
These should probably be handled at the keymap level, rather than at the kb level

Reply:
I will remove this. 

But we got ROTATIONAL_TRANSFORM_ANGLE set in 
https://github.com/Schievel1/qmk_firmware_dm_r_track/blob/54b2ede6f8351757e35c7b38ff3b54b6b02f1a31/keyboards/handwired/tractyl_manuform/5x6_right/config.h#L21
shouldn't this be removed either?